docs/source/conf.py

```diff
@@ -201,6 +201,9 @@
 # to ignore or allow certain links.
 html_extra_path = [str(here / "robots.txt")]
 
+# Likewise for search console verification
+html_extra_path = [str(here / "googlebce3ad2fda582ae8.html")]
```

Comment:
Oh, it was as easy as that 😄 (to type - not to learn...)

Reply:
Yeah, I'm a numpty. I still want to go through that exclusions list separately though as there are files in there that no longer exist.